### PR TITLE
Add mmseg2torchserve tool

### PR DESCRIPTION
**Pull request by daavoo**
This P.R. enables serving MMSegmentation models with TorchServe. A new section Model Serving has been added in useful_tools.md.

### CODE REVIEW

**Comment by xvjiarui** [comment]
Hi @daavoo 
Thanks for the nice PR! We will review it recently.

**Comment by Junjun2016** [comment]
Hi @daavoo 
Sorry for the late reply.
@AllentDan will help with the review.

**Comment by AllentDan** [comment]
Hi, it worked all fine for me except the image. We may move 3dogs_mask.png  to the folder resources [here](https://github.com/open-mmlab/mmsegmentation/tree/master/resources) for better management.

**Comment by Junjun2016** [comment]
Hi @AllentDan 
@daavoo has updated this PR.
Could you please review it again?

File: docs/useful_tools.md [hunk 1]
```diff
@@ -245,3 +245,64 @@ Examples:
...
+```python
+import matplotlib.pyplot as plt
+import mmcv
+plt.imshow(mmcv.imread("3dogs_mask.png", "grayscale"))
+plt.show()
+```
+
+You should see something similar to:
+
+![3dogs_mask](imgs/3dogs_mask.png)
```
[the remark this answers is not in the document]
Comment (daavoo) [reply docs/useful_tools.md:308]: I see

File: docs/useful_tools.md [hunk 2]
```diff
@@ -245,3 +245,64 @@ Examples:
...
+--mount type=bind,source=$MODEL_STORE,target=/home/model-server/model-store \
+mmseg-serve:latest
+```
+
+[Read the docs](https://github.com/pytorch/serve/blob/072f5d088cce9bb64b2a18af065886c9b01b317b/docs/rest_api.md) about the Inference (8080), Management (8081) and Metrics (8082) APis
+
+### 4. Test deployment
+
+```shell
+curl -O https://raw.githubusercontent.com/open-mmlab/mmsegmentation/master/resources/3dogs.jpg
```
[the remark this answers is not in the document]
Comment (AllentDan) [reply docs/useful_tools.md:291]: LGTM

**Review by AllentDan — APPROVED** [review]

**Comment by Junjun2016** [comment]
Hi @daavoo 
Could you please add some unittest?

**Comment by AllentDan** [comment]
Hi, @Junjun2016. For codes in tools other than mmseg, we may not consider unittests.

**Comment by Junjun2016** [comment]
> Hi, @Junjun2016. For codes in tools other than mmseg, we may not consider unittests.

Got it.

**Review by AllentDan — APPROVED** [review]

**Pull request merged**